kits/arm/ex_mobile_device_control.py

```diff
@@ -111,106 +116,117 @@ def get_ik(xyz_target, rot_target, ik_seed):
   # Start background logging
   if enable_logging:
     group.start_log(os.path.join(local_dir, 'logs'), mkdirs=True)
-    phone_group.start_log(os.path.join(local_dir, 'logs'), mkdirs=True)
+    #phone_group.start_log(os.path.join(local_dir, 'logs'), mkdirs=True)
```

Comment:
Do we not want to log mobile IO feedback? Just wondering. If we don't (as implied by this line being commented out), then we should delete this commented out line.